config/clients/go/CHANGELOG.md.mustache

```diff
@@ -1,20 +1,22 @@
 # Changelog
 
-## [Unreleased](https://github.com/openfga/go-sdk/compare/v{{packageVersion}}...HEAD)
+## v0.6.4
+
+### [0.6.4](https://{{gitHost}}/{{gitUserId}}/{{gitRepoId}}/compare/v0.6.3...v0.6.4) (2025-01-29)
 
 - feat: add support for `start_time` parameter in `ReadChanges` endpoint (#158)
 - fix: correctly set request level storeId in non-transactional write (#162)
 - fix: api client should set default telemetry if not specified (#160)
 
```

Comment:
Could we also add the changelog entries from openfga/go-sdk/pull/164 as part of this PR

```suggestion
- docs: replace readable names with uuid (#146) - thanks @sccalabr 
- fix: support marshaling client.ClientWriteResponse (#145) - thanks @Fedot-Compot
- fix: update client interface with store and model getter/setter (#155)
- fix: api client should set default telemetry if not specified (#160) 
- fix: Do not ignore request level storeId in non-transactional write (#162)

```